package.json

```diff
@@ -13,7 +13,8 @@
     "prebuild.prod": "npm run prebuild",
     "build.prod": "npx ng build --progress false --prod --subresource-integrity",
     "build": "npx ng build --progress false --subresource-integrity",
-    "license": "license-checker  --excludePackages dockstore-ui2@2.3.0 --csv > THIRD-PARTY-LICENSES.csv",
+    "license": "license-checker --excludePackages dockstore-ui2@2.3.0 --csv > THIRD-PARTY-LICENSES.csv",
```

Comment:
2.3.0 is in the file three times now. 
Maybe experiment with variables? https://frontend.irish/npm-config-variables

Reply:
Hm the version doesn't seem to like the variable
```
npm WARN prepare removing existing node_modules/ before installation
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "$npm_package_config_ui_version": Tags may not have any characters that encodeURIComponent encodes.
```
but works okay in the scripts section.